pkg/security/apparmor/helpers.go

```diff
@@ -43,6 +44,28 @@ func GetProfileNameFromPodAnnotations(annotations map[string]string, containerNa
 	return annotations[v1.AppArmorBetaContainerAnnotationKeyPrefix+containerName]
 }
 
+// GetProfile builds the `SecurityProfile` from the provided annotations and
+// containerName.
+func GetProfile(annotations map[string]string, containerName string) *runtimeapi.SecurityProfile {
+	profileName, ok := annotations[v1.AppArmorBetaContainerAnnotationKeyPrefix+containerName]
+
+	// TODO: if no profile is set, switch to RuntimeDefault instead of Unconfined
+	if !ok || profileName == v1.AppArmorBetaProfileNameUnconfined {
+		return &runtimeapi.SecurityProfile{
+			ProfileType: runtimeapi.SecurityProfile_Unconfined,
+		}
+	} else if profileName == v1.AppArmorBetaProfileRuntimeDefault {
```

Comment:
is there a path where it can be docker/default still?

Reply:
Probably, I added a `|| "docker/default"` just to be sure.

Reply:
I did a grep in k/k and it looks like we do not even have docker/default for AppArmor. Removed it again :see_no_evil: